det3d/datasets/argoverse/argoverse_dataset.py

```diff
@@ -0,0 +1,106 @@
+from det3d.datasets.argoverse.argoverse_common import *
+from det3d.datasets.custom import PointCloudDataset
+from det3d.datasets.registry import DATASETS
+
+
+@DATASETS.register_module
+class ArgoverseDataset(PointCloudDataset):
+
+    def __init__(self, root_path, pipeline=None, test_mode=False):
+        self._log_id_timestamps_list = ArgoverseDataset.load_logs(root_path, 1)
+        # Not using info_path but loading ground truth directly for now
+        super(ArgoverseDataset, self).__init__(
+            root_path, None, pipeline, test_mode=test_mode
+        )
+        self._num_point_features = 3  # x, y, z
```

Comment:
any reason not to use the intensity also?
The Waymo loader seems to use it:
https://github.com/hchittanuru3/CenterPoint/blob/328317fc3dbb0d55f1f89264e10fed2a59fdf226/det3d/datasets/waymo/waymo.py#L19

Here's how to load it:
```python
def load_ply(ply_fpath: _PathLike) -> np.ndarray:
	"""Load a point cloud file from a filepath.
	Args:
		ply_fpath: Path to a PLY file
	Returns:
		arr: Array of shape (N, 3)
	"""

	data = pyntcloud.PyntCloud.from_file(os.fspath(ply_fpath))
	x = np.array(data.points.x)[:, np.newaxis]
	y = np.array(data.points.y)[:, np.newaxis]
	z = np.array(data.points.z)[:, np.newaxis]
	intensity = np.array(data.points.intensity)[:, np.newaxis]

	return np.concatenate((x, y, z, intensity), axis=1)
```